README.md

```
@@ -57,34 +78,182 @@ The extension icon should now appear in your browser's toolbar.
-- Firefox: `bun run build:firefox`
+- Browser and extension version
+- Steps to reproduce the issue
+- Expected vs actual behavior
+- Console errors (if any)
 
-## License
+## 📝 License
 
-This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
\ No newline at end of file
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)